bin/improver-extract

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017 Met Office.
```

Comment:
Nitpick just because (since you'll be editing this file anyway) - should be 2018 for all files.